aptos-move/aptos-vm/src/transaction_metadata.rs

```diff
@@ -41,8 +41,11 @@ impl TransactionMetadata {
                 .map(|account_auth| account_auth.authentication_key().to_vec())
                 .collect(),
             sequence_number: txn.sequence_number(),
```

Comment:
Because we don't do the sequence number bit trick we need feature flags now in this code to prevent forking

Reply:
okay if both path are surely not going on-chain then it's fine